Updating app registration portal experience terminology to match App …

AllisonAm opened the pull request:
…Preview experience

AllisonAm commented:
@archieag

jmprieur commented on README.md:
```diff
@@ -77,17 +77,12 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
 #### Register the client app (MyDirectorySearcherApp)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
+1. In the  **Azure Active Directory** pane, click on **App registrations (Preview)** and choose **New application**.
```
Maybe remove (Preview) but add the deep link?

jmprieur commented on README.md:
```diff
@@ -77,17 +77,12 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
+1. In the  **Azure Active Directory** pane, click on **App registrations (Preview)** and choose **New application**.
 1. Enter a friendly name for the application, for example 'MyDirectorySearcherApp' and select 'Native' as the *Application Type*.
 1. For the *Redirect URI*, enter `https://<your_tenant_name>/MyDirectorySearcherApp`, replacing `<your_tenant_name>` with the name of your Azure AD tenant.
```
Actually,  things have evolved since this sample was created. The following URI might be better `https://login.microsoftonline.com/common/oauth2/nativeclient`. (for which the developer can check a check-box in the portal). But no obligation here.

AllisonAm commented:
@jmprieur I updated the Preview terminology and also updated the Redirect URI link with the one you provided. Can you confirm that all makes sense? Thank you!
